jiracli/templates.go

```diff
@@ -433,18 +441,27 @@ fields:
   assignee: {{if .fields.assignee.name}}
     emailAddress: {{ or .fields.assignee.name}}
   {{- else }}
-    emailAddress: {{.fields.assignee.emailAddress}}{{end}}{{end}}{{end}}
+    emailAddress: {{.fields.assignee.emailAddress}}
+  {{end}}{{end}}
+{{end}}
 {{- if .meta.fields.reporter}}
   reporter:
-    emailAddress: {{ if .overrides.reporter }}{{ .overrides.reporter }}{{else if .fields.reporter}}{{ .fields.reporter.emailAddress }}{{end}}{{end}}
+    emailAddress: {{ if .overrides.reporter }}{{ .overrides.reporter }}{{else if .fields.reporter}}{{ .fields.reporter.emailAddress }}{{end}}
+{{end}}
 {{- if .meta.fields.customfield_10110}}
   # watchers
-  customfield_10110: {{ range .fields.customfield_10110 }}
-    - name: {{ .name }}{{end}}{{if .overrides.watcher}}
-    - name: {{ .overrides.watcher}}{{end}}{{end}}
+  customfield_10110:
+  {{ range .fields.customfield_10110 }}
+    - name: {{ .name }}
+  {{end}}
+  {{if .overrides.watcher}}
+    - name: {{ .overrides.watcher}}
+  {{end}}
+{{end}}
```

Comment:
Keep `{{-` or `-}}` to avoid the extra blank lines.
```suggestion
  {{- range .fields.customfield_10110 }}
    - name: {{ .name }}
  {{- end}}
  {{- if .overrides.watcher}}
    - name: {{ .overrides.watcher}}
  {{- end}}
{{- end}}
```